xarray/tests/test_duck_array_ops.py

```diff
@@ -440,7 +440,9 @@ def test_argmin_max(dim_num, dtype, contains_nan, dask, func, skipna, aggdim):
             **{aggdim: getattr(da, "arg" + func)(dim=aggdim, skipna=skipna).compute()}
         )
         expected = getattr(da, func)(dim=aggdim, skipna=skipna)
-        assert_allclose(actual.drop(actual.coords), expected.drop(expected.coords))
+        assert_allclose(
+            actual.drop(list(actual.coords)), expected.drop(list(expected.coords))
```

Comment:
do we want to special case the passing of coordinate objects for  convenience?

Reply:
It's a classic special case question! https://github.com/pydata/xarray/pull/3128/files#r315000922

Reply:
I hear you, but I can also see: https://github.com/pydata/xarray/pull/3128/files#r315000922

Reply:
Fair point, but I can also see this POV: https://github.com/pydata/xarray/pull/3128/files#r315000922 ; I defer to @shoyer

Reply:
Ah wasn't aware. That makes sense.